networks/monza/monza-full-node/Cargo.toml

```diff
@@ -14,14 +14,18 @@ rust-version = { workspace = true }
 [dependencies]
 monza-executor = { workspace = true }
 m1-da-light-node-client = { workspace = true }
+protocol-unit-types = { workspace = true }
+
+movement-types = { workspace = true }
```

Comment:
Could we consolidate these two crates? `movement-types` has some common definitions as well, but it's under `util`.

Reply:
I was thinking the same when creating the new crate. I went for the protocol-units granularity because some enums are only used in the protocol-units. I will merge for now; we can consolidate them later if needed.